src/styles/main.scss

```
@@ -0,0 +1,81 @@
+$cell-size: 100px;
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+.calendar {
+  width: calc(7 * #{$cell-size} + 6px);
+  margin: 0 auto;
+  margin-top: 82px;
+  box-sizing: border-box;
+
+  &__field {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 1px;
+  }
+
+  &__day {
+    position: relative;
+    width: $cell-size;
+    height: $cell-size;
+    box-sizing: border-box;
+    border: 1px solid #000;
+    background: #eee;
+    cursor: pointer;
+    transition: transform background 0.5s;
+
+    &:hover {
+      transform: translateY(-20px);
+      background: #ffc0cb;
+    }
+
+    @for $i from 0 through 31 {
+      .calendar--month-length-#{$i} {
+        content: '#{$i}';
+        height: $cell-size;
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+        height: $cell-size;
+        font-family: Arial, sans-serif;
+        font-size: 30px;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
```

in this loop we just need to work with `::before`. Font styles you can apply for `&__day `
```suggestion
      &:nth-child(#{$i})::before {
        content: '#{$i}';
      }
```